test/ecdsa-test.js

```diff
@@ -5,6 +5,7 @@ var hash = require('hash.js');
 describe('ECDSA', function() {
   function test(name) {
     it('should work with ' + name + ' curve', function() {
+      this.timeout(5000);
```

Comment:
it takes like 2800 on saucelabs so we could lower it to 4000

On Tue, Oct 27, 2015 at 5:53 PM Fedor Indutny <notifications@github.com>
wrote:

> In test/ecdsa-test.js
> <https://github.com/indutny/elliptic/pull/58#discussion_r43193076>:
>
> > @@ -5,6 +5,7 @@ var hash = require('hash.js');
> >  describe('ECDSA', function() {
> >    function test(name) {
> >      it('should work with ' + name + ' curve', function() {
> > +      this.timeout(5000);
>
> Does it really take this long?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/indutny/elliptic/pull/58/files#r43193076>.
>